examples/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+SET(${PROJECT_NAME}_EXAMPLES
+  i-inverse-kinematics
+  overview-simple
+  #overview-urdf
+  )
+
+FOREACH(EXAMPLE ${${PROJECT_NAME}_EXAMPLES})
+  IF(BUILD_PYTHON_INTERFACE)
+    ADD_PYTHON_UNIT_TEST("py-example-${EXAMPLE}" "examples/${EXAMPLE}.py" "bindings/python")
+  ENDIF(BUILD_PYTHON_INTERFACE)
+  ADD_UNIT_TEST("example-${EXAMPLE}" ${EXAMPLE})
```

Comment:
So you have decided to treat the examples as unit tests? How about treating them separately, with a separate compilation option BUILD_EXAMPLES?

Reply:
It was a request from @nmansard . I will add the `BUILD_EXAMPLES` option.

Reply:
+1 
I like the idea of having an automatic validation of the examples.

Reply:
done in b925bce8bd54482a130893ecfdadcf7b8c009dd2